authors/apps/profiles/tests/test_profile.py

```diff
@@ -112,3 +110,27 @@ def test_update_profile(self):
             HTTP_AUTHORIZATION='Token ' + token
         )
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_get_all_authors(self):
+        """
+        Test getting a list of authors
+        :return:
+        """
+        self.email_verification(self.reg_user)
+        res = self.login(self.log_user)
+        token = res.data['token']
+        response = self.client.get(
+            reverse('authors_profile'),
+            content_type='application/json',
+            HTTP_AUTHORIZATION='Token ' + token)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
It's better practice to not only test for the status_code but also contents of the response. For example, tests to confirm that authors are indeed in the response.